models/netsuite2/intermediate/base/int_netsuite2__transaction_lines.sql

```diff
@@ -12,11 +12,37 @@ transaction_accounting_lines as (
     from {{ var('netsuite2_transaction_accounting_lines') }}
 ),
 
+accounting_books as (
+
+    select *
+    from {{ var('netsuite2_accounting_books') }}
+),
+
 joined as (
 
     select 
         transaction_lines.*,
         transaction_accounting_lines.account_id,
+        transaction_accounting_lines.accounting_book_id,
+        transaction_accounting_lines.amount,
+        transaction_accounting_lines.credit_amount,
+        transaction_accounting_lines.debit_amount,
+        transaction_accounting_lines.paid_amount,
+        transaction_accounting_lines.unpaid_amount,
+        transaction_accounting_lines.is_posting
+
+    from transaction_lines
+    left join transaction_accounting_lines
+        on transaction_lines.transaction_line_id = transaction_accounting_lines.transaction_line_id
+        and transaction_lines.transaction_id = transaction_accounting_lines.transaction_id
+
+  {% if var('netsuite2__multibook_accounting_enabled', true) %}
+    union all
```

Comment:
Could you elaborate on the purpose of this union all?

Reply:
See multibook note above. The purpose of the union is the get base book transaction onto adjustment only book with the adjustment book id.